lr_face/models.py

```diff
@@ -102,8 +125,12 @@ def embed(self,
         :param cache_dir: Optional[str]
         :return: np.ndarray
         """
-        x = image.get_image(self.resolution, normalize=True)
-        x = np.expand_dims(x, axis=0)
+        # For face_recognition model, RGB image is required.
+        if self.name == 'face_recognition':
+            x = image.get_image()
```

Comment:
How does calling `image.get_image()` ensure that the returned image is actually in RGB format? Would it not be better to code that just reverts the ordering of the color channels of `x` (i.e. the image returned by `image.get_image(...)` based on some internal variable (in this case you use `self.name == 'face_recognition'` as a condition, but this is not very generic or scalable to future architectures that also require RGB ordering).